presto-main/src/test/java/com/facebook/presto/execution/scheduler/TestPhasedExecutionSchedule.java

```diff
@@ -184,7 +186,11 @@ private static PlanFragment createBroadcastJoinPlanFragment(String name, PlanFra
         Symbol symbol = new Symbol("column");
         PlanNode tableScan = new TableScanNode(
                 new PlanNodeId(name),
-                new TableHandle(new ConnectorId("test"), new TestingTableHandle()),
+                new TableHandle(
+                        new ConnectorId("test"),
+                        new TestingTableHandle(),
+                        TestingTransactionHandle.create(),
+                        Optional.of(TestingHandle.INSTANCE)),
```

Comment:
I think we should use `Optional.empty()`, since the `TableScanNode` created doesn't have a `TableLayoutHandle`. In general, we should keep `TableScanNode.tableLayout == TableScanNode.table.layout` until we deprecate the one in `TableScanNode` (I assume this is also the PR going to do later, but I could be wrong...)